gcp-ts-gke-knative/README.md

```diff
@@ -0,0 +1,39 @@
+# gcp-ts-gke-knative
+All-in-one demo of Knative on GKE
+
+**Not intended for production use**
```

Comment:
Why is this needed?  Should we at least qualify this statement?  Is it because of Pulumi - or because of Knative?  If needed, could we soften the language?

Reply:
A little bit of both: 
* Knative + Istio are complex and we haven't tested this at all beyond the "hello world" demo.
* The current AIO stack is almost unusably slow.